docs/modules/user-guide/partials/proc_adding-event-bindings.adoc

```diff
@@ -2,11 +2,7 @@
 = Adding event bindings
 
 [role="_abstract"]
-Commands can be bound to events to form four different types of events: `preStart`, `postStart`, `preStop`, and `postStop`.
-
-.Prerequisites
-
-
+This section describes how to add an event to devfiles v2.x. An event specified in a devfile v1.x still works in a devfile v2.x.
```

Comment:
The plural "devfiles v2.x" reads very oddly to me. Should we change that to "v2.x devfiles", or has this been discussed elsewhere?

Reply:
+1 for v2.x devfiles, sounds more natural to me too

Reply:
Awesome. I've changed the order to `v2.x devfiles` and `v1.x devfiles` in all the docs. Resolving this conversation as I've addressed these concerns.